src/Symfony/Component/Notifier/Transport/AllTransport.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Transport;
+
+use Symfony\Component\Notifier\Exception\LogicException;
+use Symfony\Component\Notifier\Message\MessageInterface;
+
+/**
+ * @author Fabien Potencier <fabien@symfony.com>
+ *
+ * @experimental in 5.0
+ */
+class AllTransport implements TransportInterface
+{
+    private $transports = [];
+
+    /**
+     * @param TransportInterface[] $transports
+     */
+    public function __construct(array $transports)
+    {
+        if (!$transports) {
+            throw new LogicException(__CLASS__.' must have at least one transport configured.');
+        }
+
+        $this->transports = $transports;
+    }
+
+    public function __toString(): string
+    {
+        return implode(' all ', array_map(function (TransportInterface $transport) {
+            return (string) $transport;
+        }, $this->transports));
+    }
+
+    public function supports(MessageInterface $message): bool
+    {
+        foreach ($this->transports as $transport) {
+            if ($transport->supports($message)) {
+                return true;
+            }
+        }
+
+        return false;
+    }
+
+    public function send(MessageInterface $message): void
+    {
+        foreach ($this->transports as $transport) {
+            $transport->send($message);
```

Comment:
I think that exceptions should be catched here because it wouldn't be good if none notification will be send if first transport would be configured improperly. I mean, if you for example use Slack and Twilio and you configure wrong host for first of them, you still want to get notification in another transport :thinking:

Reply:
Instead, I've removed this class. It's not very useful as we already have a way to send a message to several transports (which does manage exceptions the way you describe it).